metpetdb_api/api/samples/lib/query.py

```diff
@@ -11,6 +11,14 @@ def sample_query(user, params, qs):
     else:
         qs = qs.filter(Q(owner=user) | Q(public_data=True))
 
+    if params.get('provenance'):
+        if(params.get('provenance')=="Public"):
+            qs = qs.filter(public_data=True)
+        elif(params.get('provenance') == "Private"):
+            qs = qs.filter(public_data=False | Q(owner=user))
+        else:
```

Comment:
@luciencd Don't need this else clause because it's taken care of by L9-L12 above.